ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -781,29 +775,32 @@ namespace ix
         _lastSendTimePoint = std::chrono::steady_clock::now();
     }
 
-    void WebSocketTransport::close()
+    void WebSocketTransport::close(uint16_t code, const std::string& reason)
```

Comment:
instead of two `close` methods you can add default value 0 to `closeWireSize` to original `close`